include/uvc_camera/tara_ros.h

```diff
@@ -11,57 +10,101 @@
 #include <fstream>
 #include <yaml-cpp/yaml.h>
 #include <string.h>
```

Comment:
Is this header file required? @akhileshmalviya . Remove if unnecessary. Otherwise it would break the package build. Or new dependencies have to be installed. Lets keep this driver as simple as possible.